pkg/storage/gcs/store.go

```diff
@@ -39,6 +39,15 @@ func clientOpts(readOnly bool, credentialFile string) []option.ClientOption {
 	return opts
 }
 
+// versionedObject wraps *BucketHandle.Object
+// towards canonicalization of object names.
+func versionedObject(
```

Comment:
@ransomw1c  when we last talked about this, we all agreed that this strategy was regarded as bringing too much change compared to what was really needed. TL, DR: just drop this and keep your original idea (adding KeyVersions only).

Reply:
i'll back out these changes momentarily.

it is an internal implementation detail that need be resolved before this request lands, not something that need block percolating finishing touches to ui.

Reply:
.. whoops, this and all `pkg/storage` changes probably belong in #419 .

i'll make the change in that patch, then, once it's landed, i'll be sure to catch on rebase to this branch.